tests/testthat/test-btw.R

```diff
@@ -10,7 +10,7 @@ test_that("btw() preserves order", {
   expect_snapshot(cli::cat_line(btw(two, one)))
 })
 
-test_that("btw() with prompt stings", {
+test_that("btw() can include prompt stings", {
```

Comment:
```suggestion
test_that("btw() can include prompt strings", {
```

Not due to this PR, but:

Reply:
haha prompt stings from tiny prompt injections